### PR TITLE
Tweaks to the devtools page

### PR DESCRIPTION
**Pull request by lukebakken**
* Remove some very out-of-date projects
* Indicate which projects have official support

### CODE REVIEW

File: driver.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def translate_path(self, path):
     if len(sys.argv) > 1:
         site_mode = sys.argv[1]
 
-    addr = ("localhost", 8191)
```
Comment (michaelklishin) [inline driver.py:69]: We intentionally switched to localhost after #1403

Comment (lukebakken) [reply driver.py:69]: Ha! I forgot about that. I'll switch it back and will just manually change it when I need to.

File: site/devtools.md [hunk 2]
```diff
@@ -194,8 +193,8 @@ Miscellaneous projects:
 
 ## <a id="erlang-dev" class="anchor" href="#erlang-dev">Erlang</a>
 
- * [RabbitMQ Erlang client guide](erlang-client-user-guide.html)
- * [RabbitMQ Stream Erlang client](https://gitlab.com/evnu/lake)
+ * &#x2713; [RabbitMQ Erlang client](erlang-client-user-guide.html)
+ * &#x2713; [RabbitMQ Stream Erlang client](https://gitlab.com/evnu/lake)
```
Comment (scudelletti) [inline site/devtools.md:197]: This Client doesn't seem official since its copyrights and organization don't seem to be related with the RabbitMQ.

Comment (lukebakken) [reply site/devtools.md:197]: You're correct, please feel free to open a PR to remove the checkmark.
